### DISCUSSION TITLE
Import of excel to add records to database

### DISCUSSION THREAD

**Discussion by NoahNxT** [orchidsoftware/platform#1664] [Q&A]
Hi,
I'm looking for a way to import a .xlsx or .csv.
I'm using the Upload component in a form, but it looks like it only supports images? Is there a way to upload files and process them? Because I would like to read the excel files and send the information records to my database.

**Comment by EeKay** [comment]
Hey @tabuna I've *bumped* into this also: can we somehow use the upload component to upload CSV file for importing its contents only? 

Or do we have to build this custom and work around Orchid for this?

**Reply by tabuna** [reply]
Hi @EeKay If you need to upload a file for further processing, not storage. 
Then in my opinion the best thing is to just specify the corresponding input:

```php
Input::make('import')
    ->type('file')
    ->accept('.csv')
    ->title('File input example')
```

The file will not be attached and will only be received in a request:

```php
public function import(Request $request) {
    $file = $request->file('import');
    //...
}
```

**Comment by EeKay** [comment]
Thanks @tabuna! This was exactly what I was looking for (totally overlooked file input type for this).
